code/index.html

```diff
@@ -2,29 +2,147 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Karla:ital,wght@0,200;0,600;1,200&family=Permanent+Marker&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/025421cb6e.js" crossorigin="anonymous"></script>
+    <link rel='icon' href='favicon.ico' type='image/x-icon'/>
+    <link rel="stylesheet" href="./style.css">
+    <title>YOGAone</title>
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    
+    <header class="hero-header">
+      <video class="hero-video" controls autoplay loop>
```

Comment:
Maybe omit the `controls` attribute so that it just autoplays, and possibly find a smaller file 😇